department-of-veterans-affairs/caseflow
# Disable save button when submitting changes to AOD

https://dsva.slack.com/archives/C3EAF3Q15/p1596044684201800

Related to https://github.com/department-of-veterans-affairs/dsva-vacols/issues/128

### Description

We are seeing duplicate AOD records in production. Some of these look like they are related to double submitting a form from the timestamps of when they were created.

### Test Plan

1. Login as BVAAABSHIRE
2. Navigate to a docket with hearings
3. Change the AOD to "Granted", and the AOD reason to anything
4. Click the save button as many times as possible, **it should be disabled after the first time you click it**
5. Ensure that only one success message displays
